canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -62,6 +63,9 @@
 		.bx--radio-button-wrapper > label {
 			justify-content: left; // override carbon style to center justify radio-button labels
 			align-items: center;
+			span:nth-child(2) {
+				padding-bottom: $spacing-01;
```

Comment:
Actually this will be fixed if you remove these styles that modify the Carbon styles. No additional styles are needed to "fix" this
![image](https://user-images.githubusercontent.com/1628100/204417811-84f754fd-4510-4c94-ba8a-693c97e8603a.png)